data-prepper-plugins/aggregate-processor/README.md

```diff
@@ -98,7 +98,7 @@ While not necessary, a great way to set up the Aggregate Processor [identificati
     The values in a list are merely appended, so there can be duplicates. 
 
 ### <a name="count"></a>
-* `count`: Count Events belonging to the same group and generate a new event with values of the identification keys and the count, indicating the number of events. All Events that make up the combined Event will be dropped.
+* `count`: Count Events belonging to the same group and generate a new event with values of the identification keys and the count, indicating the number of events. All Events that make up the combined Event will be dropped. One of the events is added as exemplar. If the aggregation is done on traces, then traceId and spanId are included in the exemplar, otherwise, spanId and traceId would be null.
```

Comment:
GitHub appears to be reporting some sort of formatting error. Is the Markdown correct?